crates/block-producer/src/debugger.rs

```diff
@@ -65,10 +65,11 @@ pub async fn build_mock_transaction(
             return Ok(vec![]);
         }
         // parse dep group
-        let cell = rpc_client
-            .get_cell(dep.out_point())
-            .await?
-            .ok_or_else(|| anyhow!("can't find dep group cell"))?;
+        let cell = match rpc_client.get_cell(dep.out_point()).await? {
+            Some(cell) => Some(cell),
+            None => rpc_client.get_cell_from_mempool(dep.out_point()).await?,
```

Comment:
Do we need to implement this in the rpc layer? If get_cell return null, we try to query mem pool, what do you think?